test/bench/document_test.ts

```diff
@@ -61,6 +61,21 @@ const tests = [
       }
     },
   },
+  {
+    name: 'Document#object',
+    run: (): void => {
+      for (let i = 0; i < 100; i++) {
+        const doc = Document.create<{ k1: string }>('test-doc');
+        doc.update((root) => {
+          root.k1 = 'v1';
+          assert.equal(`{"k1":"v1"}`, root.toJSON?.());
```

Comment:
Hi, @parkeunae In my view, we can write `toJSON!()` instead of root.toJSON?.() :)